### PR TITLE
citra_qt: Resized main config window

### PR DESCRIPTION
**Pull request by PizzicatoWolf**
Hello there! This is my first pull request. I thought I could do some work on the front-end so I decided to resize the default configuration layout width. It looks too wide and It looks much nicer when it's not so stretched out. Let me know if this is all good. :)

Original
original

Fixed
fixed

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3336)

### CODE REVIEW

File: src/citra_qt/configuration/configure.ui [hunk 1]
```diff
@@ -34,15 +34,15 @@
...
+     <widget class="ConfigureGraphics" name="graphicsTab">
+      <attribute name="title">
+       <string>Graphics</string>
+      </attribute>
+     </widget>
      <widget class="ConfigureAudio" name="audioTab">
-       <attribute name="title">
-         <string>Audio</string>
-       </attribute>
+      <attribute name="title">
```
Comment (BreadFish64) [inline src/citra_qt/configuration/configure.ui]: Why is this changed?

File: src/citra_qt/configuration/configure.ui [hunk 2]
```diff
@@ -34,15 +34,15 @@
...
       </attribute>
      </widget>
-      <widget class="ConfigureGraphics" name="graphicsTab">
-        <attribute name="title">
-          <string>Graphics</string>
-        </attribute>
-      </widget>
+     <widget class="ConfigureGraphics" name="graphicsTab">
+      <attribute name="title">
+       <string>Graphics</string>
```
Comment (BreadFish64) [inline src/citra_qt/configuration/configure.ui]: Why is this changed?

File: src/citra_qt/configuration/configure.ui [hunk 3]
```diff
@@ -17,7 +17,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>
```
Comment (BreadFish64) [inline src/citra_qt/configuration/configure.ui]: Why is this changed?

File: src/citra_qt/configuration/configure.ui (hunk 1 above)
Comment (PizzicatoWolf) [reply src/citra_qt/configuration/configure.ui]: This is unintentionally changed by Qt Designer. I'll submit a new commit with this fixed.

Comment (PizzicatoWolf) [reply src/citra_qt/configuration/configure.ui]: Just took a look at the original file and found that the rest was using one space indents except for this section which Qt Designer fixed.

File: src/citra_qt/configuration/configure.ui (hunk 2 above)
Comment (wwylele) [reply src/citra_qt/configuration/configure.ui]: indentation fix, I guess
